modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/entry/ListEntries.es.js

```diff
@@ -27,7 +26,7 @@ import {getLocalizedUserPreferenceValue} from '../../utils/lang.es';
 import NoPermissionEntry from './NoPermissionEntry.es';
 import {buildEntries, navigateToEditPage} from './utils.es';
 
-export default withRouter(({history}) => {
+export default ({history}) => {
```

Comment:
Done, @albinojunior! Thanks for you review.

Reply:
Hey @interaminense, what do you think about export the component as a named module?

Just for make easy to found the components in your files and the import. It's just a suggestion, because we had some problems with auto import of unnamed modules. It's happen because of the files suffix `es.js`.

Something like:

`import ListEntriesEs from './ListEntries.es'`

Reply:
Alright, no problem @albinojunior. I can fix this. Thanks!